### ISSUE TITLE
Add new page id and token to bot service Facebook channel

### ISSUE THREAD

**Issue by josephktcheung** [Azure/azure-cli#7564]
**Is your feature request related to a problem? Please describe.**
I don't know how to add new Facebook page id and token to an existing bot channel registration.

**Describe the solution you'd like**
From azure cli, it should describe how to update Facebook channel with new page id and token, currently it only has `az bot facebook create` which creates a new channel.

**Describe alternatives you've considered**

**Additional context**
Base on https://github.com/Microsoft/BotBuilder/issues/1495#issuecomment-408584611, we can have multiple pages connected to a bot channel registration but it's unclear how to do it using the cli

**Comment by carlosscastro** [comment]
Hello @josephktcheung we only support a single page in the CLI. The bot framework supports multiple but the CLI supports only one for now. 

If you want to replace it, you can just delete and create again with your new page.

We are planning to support multiple facebook pages for an upcoming release, but at this point we can't tell how soon what will be. I'll update when we have a concrete date!

**Comment by carlosscastro** [comment]
In the meantime, you could call the ARM API that does support multiple facebook endpoints. Basically calling PUT channel, and passing a FacebookChannelProperties with multiple pages, see here for the swagger:

https://github.com/Azure/azure-rest-api-specs/blob/master/specification/botservice/resource-manager/Microsoft.BotService/preview/2018-07-12/botservice.json#L1553

**Comment by stevengum** [comment]
Let's investigate this for the 4.4 release.

**Comment by HellDevil** [comment]
Any news regarding this issue ?

**Comment by PuTaTus** [comment]
Nope, I tried to contact microsoft regarding this issue. They told me that the feature is yet to be added.

**Comment by MalekBouba** [comment]
Any news, we desperately need this! @azure-sdk @carlosscastro  @stevengum
